### ISSUE TITLE
control speakers using Amazon Echo Alexa / Google Home voice support

### ISSUE THREAD

**Issue by SteevoTabasco** [AetherThings/AetherCone#18]
any way to control Aether Cone Speakers with an Amazon Echo or Google Home Smart assistant

**Comment by ammamoore** [comment]
Yes, you can control Aether Cone Speakers using Bluetooth if these speakers are certified and support Advanced Audio Distribution Profile (A2DP) and Audio / Video Remote Control Profile (AVRCP)

connect Bluetooth speaker to your echo speaker

open Alexa app->setting->Bluetooth > Pair a New Device 
echo device discovers Bluetooth speakers that you can see in Alexa app (in the list of available devices)
Select your Bluetooth speaker ->within minutes your echo device connects to Bluetooth speaker and Alexa tells you if the connection is successful. Select continue tab on Alexa app now you devices are paired and you can enjoy the music. Still, facing issue contact [Amazon Echo Speaker Support ](https://echoalexasupport.com/troubleshoots/echo-music-issues/).

**Comment by ammamoore** [comment]
I think you are talking about multiroom music playback with Amazon Echo or google home, I don’t know about google home, however amazon echo provide multiroom music playback support and you can add as many devices. Multiroom audio playback is available to Echo owners in the US, UK, and Germany on Echo, Echo Dot, and Echo Show devices.The process is as below
Amazon Alexa App >> Settings>>open pairing mode
Now turn on your Bluetooth speakers and within one minute you will all devices will pop under set up device section just click on each device to connect and you are done.

However, it is advised that name each speaker different and then groups them under multiroom music so that your voice command do not interfere with each other Like you can name one speaker an echo kitchen and other echo restroom and so on and your voice command would be as per the there name.

**Comment by SteevoTabasco** [comment]
I'm not sure that is correct - it didn't work when I tried it & then  a google search has provided a couple of article stating the multi room music feature is only currently achieved by using the inbuilt speaker on multiple echo devices - not external Bluetooth speakers like I have - fingers crossed this feature get enabled / expanded ...
